frontend/src/metabase/admin/databases/containers/DatabaseEditApp.tsx

```diff
@@ -215,4 +239,5 @@ export default _.compose(
   title(
     ({ database }: { database: DatabaseData }) => database && database.name,
   ),
+  withRouter,
```

Comment:
This is unnecessary, there are 2 new props for this component
1. `route`: this is from the `Route` component.
2. `onChangeLocation`: this is from a new key in `mapStateToProps`

I tried removing this line and the tests still passed. Also I briefly did manual testing it also looked fine.

Reply:
Perfect, thanks!
0cfa7b94335fc07f27ac4248dfbb713615c0064b